salt/config/__init__.py

```diff
@@ -43,9 +43,10 @@
 
 try:
     import psutil
-    psutil.virtual_memory
+    if not hasattr(psutil, 'virtual_memory'):
+        raise ImportError('Version of psutil too old.')
```

Comment:
@cachedout I am deeply relaxed. I see a lot of HAS_PSUTIL in salt code. I ask myself if there is simpler and cleaner solution, to have all these "IFs" in one place. Conditionless code is impossible, but code with less conditions not.